### ISSUE TITLE
Training / reference / labelled / unlabelled set

### ISSUE THREAD

**Issue by andycasey** [andycasey/AnniesLasso#53]
Audit terminology.

**Comment by andycasey** [comment]
OK based on Ness (2015a) I am going to rename the labelled set to the reference set, and call the unlabelled set the test set. The training and validation sets are disjoint subsets of the reference set.

**Comment by davidwhogg** [comment]
yes!
